packages/documentation/src/pages/forms/bypassing-schemaform.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Bypassing the SchemaForm
+---
+
+# Bypassing the `SchemaForm`
+The schema-based forms library can be used to quickly build simple form applications. It handles a lot of common things under the hood such as validation, routing, and state management.
+
+For more complicated pages, however, it can be very difficult to work with the `schema` and `uiSchema` to do what you need to. This is where bypassing the `SchemaForm` comes in.
+
+## What does "bypassing the `SchemaForm`" mean?
+The schema-based forms library in `vets-website` renders a component called `SchemaForm`. This component uses `react-jsonschema-form` to render form fields using the `schema` and `uiSchema` for a page in the `formConfig`. **Bypassing the `SchemaForm` component means you can render a form page using plain React components.** No `schema`. No `uiSchema`.
+
+You may bypass `SchemaForm` for one page at a time. This means you can use `schema` and `uiSchema` when it's helpful, and ignore it when it isn't.
+
+## How to bypass `SchemaForm`
+In your form's `formConfig` for a given page, you may supply `CustomPage` and `CustomPageReview` properties which contain React components.
+
+**Example**
+```js
+// form/config.js
+import CustomPage from './CustomPage'; // React component
+import CustomPageReview from './CustomPageReview'; // React component
+
+export const formConfig = {
+  // Some config here...
+  chapters: {
+    chapterOneName: {
+      // Chapter config here...
+      pages: {
+        mySchemalessPage: {
+          path: 'my-schemaless-page',
+          title: 'Bypassing the SchemaForm',
+          CustomPage,
+          CustomPageReview,
+          schema: { // This does still need to be here or it'll throw an error
```

Comment:
Is there any way to make `schema` optional?

Reply:
There probably is, but I didn't dig into what all we'd need to change to make this happen. When I tried omitting it, it threw some errors, so I tried leaving it functionally empty and that did the trick.

My hunch is that changing the library to make `schema` and `uiSchema` truly optional would be kind of a lot of work, but I don't know that for sure, to be fair.